src/MathProg/formulation.jl

```diff
@@ -119,21 +119,30 @@ set_matrix_coeff!(
 
 """
     setvar!(
-        formulation::Formulation, name::String, duty::Duty{Variable};
-        cost::Float64 = 0.0,
-        lb::Float64 = 0.0,
-        ub::Float64 = Inf,
-        kind::VarKind = Continuous,
-        inc_val::Float64 = 0.0,
-        is_active::Bool = true,
-        is_explicit::Bool = true,
-        moi_index::MoiVarIndex = MoiVarIndex(),
+        formulation, name, duty;
+        cost = 0.0,
+        lb = 0.0,
+        ub = Inf,
+        kind = Continuous,
+        inc_val = 0.0,
+        is_active = true,
+        is_explicit = true,
+        moi_index = MoiVarIndex(),
         members = nothing,
         id = generatevarid(duty, form)
     )
 
 Create a new variable in a formulation with given name and duties.
-Other arguments are facultative.
+Other arguments are facultative:
+ - `cost`: cost, the value the variable assumes
+ - `lb`: lower bound
+ - `ub`: upper bound
+ - `kind`: kind, which can be `Continuous`, `Binary` or `Integ`
+ - `is_active`: `true` if the variable is used in the formulation, `false` to fake its deletion
+ - `is_explicit`: `false` if the variable can be represented as the linear combination of explicit ones
+ - `moi_index`: index of the MOI representation of a Coluna Variable
+ - `members`: variable coefficient at each constraint
+ - `id`: id, identifies the variable
```

Comment:
```suggestion

Following keyword arguments allow the user to set additional information about the new variable :
 - `cost`: cost of the variable in the objective function
 - `lb`: lower bound of the variable
 - `ub`: upper bound of the variable
 - `kind`: kind which can be `Continuous`, `Binary` or `Integ`
 - `is_active`: `true` if the variable is used in the formulation, `false` otherwise
 - `is_explicit`: `true` if the variable takes part to the formulation, `false` otherwise (e.g. a variable used as a shortcut for calculation purposes)
 - `members`: a dictionary `Dict{ConstrId, Float64}` that contains the coefficients of the new variable in the constraints of the formulation (default coefficient is 0).
```

I removed `moi_index` and  `id`  because they should not be changed by a user